src/containers/community/CommunityPosts.js

```diff
@@ -42,10 +43,10 @@ const getFeedModule = (community, currentUser, moduleChoice) => {
 
   if (showPopularSkills && showPostPrompt) {
     module.component = moduleChoice
-    ? <PopularSkillsModule community={community} />
+    ? <PopularSkillsModule slug={community.slug} />
```

Comment:
ahhhh... i see. the way we're doing this feels like a React antipattern. do you mind if i poke around this and try to refactor?